modules/administration-guide/partials/proc_setting-up-an-application-link-on-the-bitbucket-server.adoc

```diff
@@ -44,8 +44,22 @@ openssl rand -base64 24 > bitbucket-shared-secret
 
 . Paste the content of the `bitbucket-consumer-key` file as the *Consumer key*.
 
+. Encode the content of the `bitbucket-consumer-key` file to Base64 for use when applying the application link Secret for the Bitbucket Server:
++
+[subs="+quotes,+attributes,+macros"]
+----
+$ echo -n '__<bitbucket-consumer-key file content>__' | base64
+----
+
```

Comment:
I suggest moving this new step as the first step to modules/administration-guide/partials/proc_applying-an-application-link-secret-for-the-bitbucket-server.adoc.

Reply:
Yes, and there we have these as the last steps.
However, this procedure currently consists of 18 steps, which is very long, so the ideal solution is to move these two new steps to the other file where they are needed. However, if you want to keep these two steps in this file, then my second-best suggestion is to move these two new steps to be the last steps of the current procedure.